### ISSUE TITLE
Typographical error in doc/core/howto/trial.xhtml

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5625]
|[<img alt="ashfall's avatar" src="https://avatars.githubusercontent.com/u/1138873?s=50" width="50" height="50">](https://github.com/ashfall)| @ashfall reported|
|-|-|
|Trac ID|trac#5625|
|Type|defect|
|Created|2012-04-09 13:31:39Z|

"More good pratices" should be corrected to "More good practices".

Attachments:

* [5625-fix-typo.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/30c/30c5a56d51ce7c0c0cacc3b412f7f94f8fcd06ee/cc6049b0497b0a5f066f3419490fef01ac680eee.patch) (394 bytes) - added by ashfall on 2012-04-09 13:52:12Z - Typo corrected.
* [5625-topfile.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/30c/30c5a56d51ce7c0c0cacc3b412f7f94f8fcd06ee/9efacb18a64978b9252690af8bd681981bb7308c.patch) (299 bytes) - added by ashfall on 2012-04-09 13:52:38Z - Added topfile

<details><summary>Searchable metadata</summary>

```
trac-id__5625 5625
type__defect defect
reporter__ashfall ashfall
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1333978299000000 1333978299000000
changetime__1334014191000000 1334014191000000
version__None None
owner__exarkun exarkun

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="ashfall's avatar" src="https://avatars.githubusercontent.com/u/1138873?s=50" width="50" height="50">](https://github.com/ashfall)|@ashfall removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun set owner to @exarkun<br>@exarkun set status to `assigned`|
|-|-|

Thanks.  The fix looks good.  I think the news fragment should be a "misc" though, since the change doesn't substantially enhance the value of the trial howto.

One other random note - trac seems to use spaces to separate keywords, not commas.  If you use "documentation, review" then it thinks the ticket has two keywords, "documentation," and "review".

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set status to `closed`|
|-|-|

(In [34159]) Fix a typo in the trial testing howto

Author: ashfall
Reviewer: exarkun
Fixes: [#5625](https://github.com/twisted/twisted/issues/5625)
